<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZDialogBase</class>
 <widget class="QDialog" name="ZDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZDialogBase</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wgt_main" native="true">
     <property name="styleSheet">
      <string notr="true">#wgt_main {
	border: 5px;	
	border-image: url(:/images/bg_dlg_main.png) 5;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="wgt_title" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#wgt_title {border-image: url(:/images/bg_title.png);}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_title">
           <property name="text">
            <string notr="true">Title</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QPushButton" name="btn_min">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#btn_min {border-image: url(:/images/btn_dlg_min.png);}
#btn_min:hover {border-image: url(:/images/btn_dlg_min_hover.png);}
#btn_min:pressed {border-image: url(:/images/btn_dlg_min_pressed.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QPushButton" name="btn_max">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#btn_max {border-image: url(:/images/btn_dlg_max.png);}
#btn_max:hover {border-image: url(:/images/btn_dlg_max_hover.png);}
#btn_max:pressed {border-image: url(:/images/btn_dlg_max_pressed.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QPushButton" name="btn_close">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#btn_close {border-image: url(:/images/btn_dlg_close.png);}
#btn_close:hover {border-image: url(:/images/btn_dlg_close_hover.png);}
#btn_close:pressed {border-image: url(:/images/btn_dlg_close_pressed.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wgt_content" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_min</sender>
   <signal>clicked()</signal>
   <receiver>ZDialogBase</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>ZDialogBase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
